ddl/ddl.go

```diff
@@ -491,6 +491,11 @@ func (d *ddl) doDDLJob(ctx sessionctx.Context, job *model.Job) error {
 		}
 
 		if historyJob.Error != nil {
+			if historyJob.State == model.JobStateRollbackDone && !historyJob.Error.Equal(errCancelledDDLJob) {
```

Comment:
Why add the condition of `!historyJob.Error.Equal(errCancelledDDLJob)`?

Reply:
Does cancel ddl job will also return the error?

Reply:
Thanks, I had changed it~